<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::ImageView</class>
 <widget class="QDialog" name="ns::ImageView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Image Viewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widgetInner {
	border: 1px solid #4E4945;
	border-radius: 4px;
	
}

/* css of caption */
#frameCaption {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #0078c7, stop: 1.0 #016aae);
/*	border-top: 1px solid #4E4945;
	border-right: 1px solid #4E4945;
	border-left: 1px solid #4E4945;*/
	/*border-bottom: 1px solid #4E4945;*/
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
}

#frameCaption #labelCaptionLogo {
	background-image: url(:/images/ui/logo.png);
	background-repeat: none;
	background-position: center;
	max-width: 32px;
	min-width: 32px;
	max-height: 32px;
	min-height: 32px;
}

#frameCaption #labelTitle {
	font-size: 13px;
	font-weight: bold;
	color: #ffffff;
}

#frameCaption #frameSystemMenu {
	border: 0px;
}

#frameCaption #frameSystemMenu #sysPanelContainer {
	border: 0px;
}

#frameCaption #pushButtonClose {
	border-image: url(:/images/ui/caption_close.png);
}

#frameCaption #pushButtonClose:hover {
	border-image: url(:/images/ui/caption_close_hover.png);
}

#frameCaption #pushButtonClose:pressed {
	border-image: url(:/images/ui/caption_close_pressed.png);
}

#frameCaption #pushButtonMaxmize {
	border-image: url(:/images/ui/remote_video_maxmize.png);
}

#frameCaption #pushButtonMaxmize:hover {
	border-image: url(:/images/ui/remote_video_maxmize_hover.png);
}

#frameCaption #pushButtonMaxmize:pressed {
	border-image: url(:/images/ui/remote_video_maxmize_pressed.png);
}

#frameCaption #pushButtonNormal {
	border-image: url(:/images/ui/remote_video_minmize.png);
}

#frameCaption #pushButtonNormal:hover {
	border-image: url(:/images/ui/remote_video_minmize_hover.png);
}

#frameCaption #pushButtonNormal:pressed {
	border-image: url(:/images/ui/remote_video_minmize_pressed.png);
}

#frameCaption #pushButtonMinimize {
	border-image: url(:/images/ui/caption_minimize.png);
}

#frameCaption #pushButtonMinimize:hover {
	border-image: url(:/images/ui/caption_minimize_hover.png);
}

#frameCaption #pushButtonMinimize:pressed {
	border-image: url(:/images/ui/caption_minimize_pressed.png);
}

#widgetInner #scrollArea {
	border: none;
}


</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetInner" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <widget class="QFrame" name="frameCaption">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelCaptionLogo">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTitle">
           <property name="text">
            <string>YesPo Service Client</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonMinimize">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>36</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonMaxmize">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>36</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonNormal">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>36</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="sysPanelContainer">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="sysPanel" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButtonClose">
                 <property name="minimumSize">
                  <size>
                   <width>35</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>35</width>
                   <height>36</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>9</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>838</width>
           <height>528</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="ns::ImageLabel" name="labelImage">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ns::ImageLabel</class>
   <extends>QLabel</extends>
   <header>imagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="translator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ns::ImageView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMinimize</sender>
   <signal>clicked()</signal>
   <receiver>ns::ImageView</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
